src/pubsub/peer-streams.js

```diff
@@ -43,35 +46,41 @@ class PeerStreams extends EventEmitter {
     this.id = id
     /**
      * Established protocol
+     *
      * @type {string}
      */
     this.protocol = protocol
     /**
      * The raw outbound stream, as retrieved from conn.newStream
+     *
      * @private
-     * @type {DuplexIterableStream}
+     * @type {null|DuplexIterableStream}
      */
     this._rawOutboundStream = null
     /**
      * The raw inbound stream, as retrieved from the callback from libp2p.handle
+     *
      * @private
-     * @type {DuplexIterableStream}
+     * @type {null|DuplexIterableStream}
      */
     this._rawInboundStream = null
     /**
      * An AbortController for controlled shutdown of the inbound stream
+     *
      * @private
-     * @type {typeof AbortController}
+     * @type {null|AbortController}
      */
     this._inboundAbortController = null
     /**
      * Write stream -- its preferable to use the write method
-     * @type {import('it-pushable').Pushable<Uint8Array>>}
+     *
+     * @type {null|import('it-pushable').Pushable<Uint8Array>}
```

Comment:
don't import inline, this makes some tools confused and the .d.ts generated more complex